satpy/readers/pmw_channels_definitions.py

```diff
@@ -0,0 +1,415 @@
+    advantageous (giving better NeDT) to sense in a band both right and left of
+    the central absorption frequency. But to avoid (CO2) absorption lines
+    symmetrically positioned on each side of the main absorption band it is
+    common to split the side bands in two 'side-side' bands.
+
+    This is needed because of this bug: https://bugs.python.org/issue41629
+
+    """
+
+    central: float
+    side: float
+    sideside: float
+    bandwidth: float
+    unit: str = "GHz"
+
+
+class FrequencyQuadrupleSideBand(FrequencyQuadrupleSideBandBase):
+    """The frequency quadruple side band class.
+
+    The elements of the quadruple-side-band type frequency band are the
+    central frquency, the relative (main) side band frequency (relative to the
+    center - left and right), the sub-side band frequency (relative to the
+    offset side-band(s)) and their bandwidths. Optionally a unit (defaults to
+    GHz) may be specified. No clever unit conversion is done here, it's just
+    used for checking that two ranges are comparable.
+
+    Frequency Quadruple Side Band is supposed to describe the special type of
+    bands commonly used in temperature sounding from Passive Microwave
+    Sensors. When the absorption band being observed is symmetrical it is
+    advantageous (giving better NeDT) to sense in a band both right and left of
+    the central absorption frequency. But to avoid (CO2) absorption lines
+    symmetrically positioned on each side of the main absorption band it is
+    common to split the side bands in two 'side-side' bands.
+
+    """
+
+    def __eq__(self, other):
+        """Return if two channel frequencies are equal.
+
+        Args:
+            other (tuple or scalar): (central frq, side band frq, side-side band frq,
+            and band width frq) or scalar frq
+
+        Return:
+            True if other is a scalar and min <= other <= max, or if other is
+            a tuple equal to self, False otherwise.
+
+        """
+        if other is None:
+            return False
+        if isinstance(other, numbers.Number):
+            return other in self
+        if isinstance(other, (tuple, list)) and len(other) == 4:
+            return other in self
+        return super().__eq__(other)
+
+    def __ne__(self, other):
+        """Return the opposite of `__eq__`."""
+        return not self == other
+
+    def __lt__(self, other):
+        """Compare to another frequency."""
+        if other is None:
+            return False
+        return super().__lt__(other)
+
+    def __gt__(self, other):
+        """Compare to another frequency."""
+        if other is None:
+            return True
+        return super().__gt__(other)
+
+    def __hash__(self):
+        """Hash this tuple."""
+        return tuple.__hash__(self)
+
+    def __str__(self):
+        """Format for print out."""
+        return "{0.central} {0.unit} ({0.side}_{0.sideside}_{0.bandwidth} {0.unit})".format(self)
```

Comment:
Is the representation using `_` based on a community standard?  It yields formats that may be confusing, because underscores in number literals have a meaning in Python.  For example, `322_50_36` means `3225036`, but I think that is not intended here.  Therefore, using `_` could be confusing or misleading.

Reply:
```suggestion
        return "f{self.central} {self.unit} ({self.side}_{self.sideside}_{self.bandwidth} {self.unit})"
```

Reply:
Thanks, I have revised this a bit, and have a new suggestion!